test/gitea_test.exs

```diff
@@ -103,13 +96,18 @@ defmodule GiteaTest do
   test "Gitea.clone clones a known remote repository gitea on Fly.io" do
     org = "nelsonic"
     repo = "public-repo"
+    # first delete if already exists:
+    Useful.empty_dir_contents(@git_dir)
+
+    # get url for cloning
     git_repo_url = Gitea.Helpers.remote_url_ssh(org, repo)
 
-    assert {:ok, path} = Gitea.clone(git_repo_url)
+    # clone should work both in mock: true and mock: false
+    assert {:ok, _path} = Gitea.clone(git_repo_url)
     # assert path == Gitea.Helpers.local_repo_path(repo)
 
     # Clean up (but don't delete the remote repo!!)
-    delete_local_directory("public-repo")
+    Useful.empty_dir_contents(@git_dir)
```

Comment:
Figured this was a more fool-proof way of clearing out the `tmp/` dir ... https://github.com/dwyl/useful/pull/15 🤞